# is the body found by the mocap system
bool found_body

#-------------------------#
# time
float64 time

#-------------------------#
# State of the Quad

# linear position
float64 x
float64 y
float64 z

# linear velocity
float64 vx
float64 vy
float64 vz

# angular position
float64 pitch
float64 roll
float64 yaw
